l10n_cl_etd_xerox/models/stock_picking_batch.py

```diff
@@ -7,6 +7,11 @@
 class PickingBatch(models.Model):
     _inherit = "stock.picking.batch"
 
+    # TODO: move this this fieldservice_route
+    company_id = fields.Many2one(
+        'res.company',
+        default=lambda s: s.env.user.company_id)
+
```

Comment:
@dreispt Added to `fieldservice_vehicle_stock` in https://github.com/OCA/field-service/pull/553/files#diff-5e43b7d68903d4ad8c35918261bf65c5R11